intranet/templates/eighth/admin/transfer_students.html

```diff
@@ -30,6 +30,16 @@
                 {% endif %}
             {% endcomment %}
         </form>
+    {% elif moved_students %}
+        <p>
+            The following students were already signed up for activities on {{ dest_act.block }}.
+            These signups have been replaced with {{ dest_act }}:
```

Comment:
It might be nice to reiterate whether the students have been notified via email, just for reference. i.e. say either "The students have been notified" or "The students have **not** been notified."